todo-app/src/TodoApp.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import TodoList from './TodoList'
```

Comment:
Another picky comment: missing a semicolon here (and a few other places). Since we're human, it's very easy to miss things like this. That's where linters come in handy.